src/libs/actions/Session.js

```diff
@@ -126,6 +127,7 @@ function fetchAccountDetails(login) {
                     validated: response.validated,
                     closed: response.isClosed,
                     forgotPassword: false,
+                    validationCodeFailedMessage: null,
```

Comment:
I'm not sure if this is the best way to handle these errors.  

It looks like this will set the translation key for the error message?

I think it would be better to make this an object with a key of `errors` rather than have an object key per error. We are using a similar strategy in other forms so it would be a good idea to align on this and keep things consistent.